### ISSUE TITLE
Pass Robert Peip's timingtest.ws

### ISSUE THREAD

**Issue by asiekierka** [FluBBaOfWard/NitroSwan#30]
https://github.com/MiSTer-devel/WonderSwan_MiSTer/tree/main/testroms/timingtest

NitroSwan is decently close; for context, Ares passes all, while Mednafen passes none.

**Comment by FluBBaOfWard** [comment]
Yeah, I can't figure out the timing of the base loop unless "in al, dx" takes 6 cycles as the documentation says...
The code is

> test_jumponly:
   mov cx,1000
align 2
repeat_jumponly:
   nop
   nop
   nop
   in al, dx
   dec cx
   jnz repeat_jumponly
   ret

nop *3 = 3
in al, dx = 6
dec cx = 1
jnz = 4
total 14

I know that adding another nop adds 1 cycle, adding another "in al, dx" adds 5 cycles, dec takes 1 cycle. I even added my own jnz test and it shows that it takes 4 cycles... Not sure how to proceed.
